api/src/main/java/org/pmiops/workbench/firecloud/FireCloudService.java

```diff
@@ -41,4 +43,6 @@
    * Retrieves all billing project memberships for the user from FireCloud.
    */
   List<BillingProjectMembership> getBillingProjectMemberships() throws ApiException;
+
+  WorkspaceACLUpdateResponseList updateWorkspaceACL(String projectName, String workspaceName, Boolean inviteUsersNotFound, List<WorkspaceACLUpdate> aclUpdates) throws ApiException;
```

Comment:
I'm pretty sure inviteUsersNotFound is always going to be false; you could remove that here probably.